float32 air_velocity
float32 air_velocity_low_pass
uint8 light_duty
bool vacuum_active
bool vacuum_power_active

# High-level filter result: Is an object attached to the tip?
bool something_on_tip
bool object_well_attached
